<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ShaderDialogClass</class>
 <widget class="QDialog" name="ShaderDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ShaderDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>320</y>
     <width>409</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>241</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="uvTab" >
    <attribute name="title" >
     <string>Uniform Variables</string>
    </attribute>
    <widget class="QWidget" name="varListLayer" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="varNameLabel" >
        <property name="text" >
         <string>Variable Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="varList" />
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>391</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="varRes" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text" >
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="glTab" >
    <attribute name="title" >
     <string>OpenGL Status</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="vpTab" >
    <attribute name="title" >
     <string>Vertex Program</string>
    </attribute>
    <widget class="QTextBrowser" name="vpTextBrowser" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fpTab" >
    <attribute name="title" >
     <string>Fragment Program</string>
    </attribute>
    <widget class="QTextBrowser" name="fpTextBrowser" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
